Equipment/EquipmentDb/Weapons/1hAxes.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
 <weapons>
+	<melee_weapon id="18737" phase="1">
+		<info name="Bone Slicing Hatchet" type="AXE" slot="1H" unique="no" req_lvl="57" item_lvl="62" quality="RARE" boe="no" icon="Inv_axe_11.png"/>
+		<dmg_range min="48" max="90" speed="1.7"/>
+		<stats>
+			<stat type="AGILITY" value="15"/>
```

Comment:
It should have 13 Agility.

Reply:
Good catch on the type-o, this should be fixed now.

Reply:
This gtg now?

Reply:
Yes. I apologize for the delay, I've been away for Christmas vacation.